server/health/health.go

```diff
@@ -0,0 +1,25 @@
+package health
```

Comment:
Do we think this warrants its own package?

Reply:
I tried putting it into the server package but then the methods didn't make any sense anymore and I had to somehow add `health` somewhere in the name – it really wasn't pretty. That's why I went with the extra package.

Reply:
Makes sense.